src/cli/index.ts

```diff
@@ -152,6 +152,16 @@ async function main(argv: string[]): Promise<void> {
       `)
     )
   }
+
+  const provider = prisma2.provider()
+
+  // MongoDB is not supported and has a custom upgrade documentation
+  if (provider === 'mongodb') {
+    return fatal(
+      `Meooowww. Custom error message with link ${gray(underline(`https://perdu.com`))}`
```

Comment:
I think we have an issue open to create some documentation about upgrading - so that would be a good link target when it exists.

Reply:
So blocked by https://github.com/prisma/docs/issues/2767

Reply:
Until then just create a fitting error message and no link - still better than nothing.